Source/MultiPlayerServer/MultiPlayerServer.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<TargetFramework>net6-windows</TargetFramework>
```

Comment:
Hi Carlo, I think the Appveyor is failing due to the code targetting .NET 6 which it doesn't support. The rest of the code is targetting .NET Framework 4.7.2 IIRC.

Reply:
Thanks César, yes, I'm aware of that. Unfortunately if I select .NET Framework 4.7.2 for this feature I get a bunch of errors when building the solution that I don't know how to eliminate.

Reply:
Given that the code is pretty simple, did you try to create a new 4.7.2 project and just copy the two source files into it?

Reply:
César, it's not as simple as that. The code contains keywords and parameter lists that aren't recognized in 4.7.2. You can see in this screenshot [](https://www.interazioni-educative.it/Varie/MPSErrors.jpg) the list of errors that I get.

Reply:
Ah, I see. Then the appveyor.yml file needs some modifications (maybe the builds need to run in VS 2022 instead of VS 2019?), but I don't know how to test that. I think targetting .NET 6 is a better option since that would allow running the MP server in Linux servers as well.

Reply:
I agree with you. I think that upgrading appveyor is something for the ORMT.

Reply:
Hi Carlo,

Just to say that James has sorting this out on our list of outstanding tasks.

Chris

Reply:
I have fixed the AppVeyor build on VS2019 (which I believe we still support) by downgrading the multiplayer server to .NET 5 (no code changes needed)

I have also updated the project metadata to align better with the existing projects

Reply:
Thanks, James.